libs/designsystem/src/lib/scss/base/_lists.scss

```diff
@@ -0,0 +1,89 @@
+$bullet-font-size: 1.3em;
+$hollow-bullet-font-size: 0.8em;
+$list-item-line-height: 1.2em;
+
+// TODO: Move to utilities/functions
+@function unitless($number) {
+  @return $number / ($number * 0 + 1);
+}
+
+// -----------------------------------------------------------------------------
+// Grid layout
+// -----------------------------------------------------------------------------
+li {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  // Nested lists
+  ol,
+  ul {
+    grid-column: 2 / -1;
+  }
+}
+
+// -----------------------------------------------------------------------------
+// Spacing
+// -----------------------------------------------------------------------------
+ol,
+ul {
+  padding-inline-start: size('xxxs'); // left
+}
+
+li {
+  &::before {
+    padding-inline-end: size('xs'); // right
+  }
+
+  // Vertical space between list items (excluding the first outermost item)
+  li,
+  & + li {
+    margin-block-start: size('xxs'); // top
+  }
+}
+
+// -----------------------------------------------------------------------------
+// Markers (bullets and numbers)
+// -----------------------------------------------------------------------------
+
+ol,
+ul {
+  list-style: none;
+  counter-reset: item;
+}
+
+// First level marker
+ul > li::before {
+  content: '\2022'; // Unicode disc ("bullet")
+  font-size: $bullet-font-size;
+  line-height: (unitless($list-item-line-height) / unitless($bullet-font-size));
+}
+
+// Second level marker
+ul ul > li::before {
+  content: '\25E6'; // Unicode circle
+  font-size: $hollow-bullet-font-size;
+  line-height: (unitless($list-item-line-height) / unitless($hollow-bullet-font-size));
+}
+
+// Third level (and beyond) marker
+ul ul ul > li::before {
```

Comment:
```suggestion
:where(ol, ul) :where(ol, ul) ul > li::before {
```

Reply:
In case you're not familiar with the `:where` pseudo-class function, it's a more concise way of writing this selector

```css
// Third level (and beyond) marker
ol ol ul > li::before,
ol ul ul > li::before,
ul ol ul > li::before,
ul ul ul > li::before
```

which _could_ also be written in Sass as

```css
ol,
ul {
  ol,
  ul {
    ul > li::before {
      content: '\25AA'; // Unicode square
    }
  }
}
```

with the essential difference, that the latter two will have a much higher specificity. `:where()` has 0 (zero) specificity

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:where